src/Components/AccountPageSubPages/PersonalLeaderBoard.jsx

```diff
@@ -20,18 +52,19 @@ export default function PersonalLeaderBoard(props) {
               <TableCell align="center">Date</TableCell>
             </TableRow>
           </TableHead>
-
-          <TableBody>
-            {scoreList.map((row, index) => (
-              <TableRow
-                key={index}
-                sx={{ "&:last-child td, &:last-child th": { border: 0 } }}
-              >
-                <TableCell align="center">{row.score}</TableCell>
-                <TableCell align="center">{row.date}</TableCell>
-              </TableRow>
-            ))}
-          </TableBody>
+          {scoreList !== undefined && scoreList.length > 0 ? (
```

Comment:
do you need !== undefined here? Or do you still want a table when the scoreList is empty?

Reply:
and scoreList.length > 0?

Reply:
had an issue where sometimes the scorelist from the api returns undefined if the link is like broken or sth

Reply:
if you just use 'scoreList ?' does that cover that issue?